### DISCUSSION TITLE
Batched only generates one line?

### DISCUSSION THREAD

**Discussion by drscotthawley** [SYSTRAN/faster-whisper#1187] [Q&A]
I find that when I run in batched inference mode, I only get one output line of transcribed results, rather than the entire transcription... 

 I'm using the syntax from the README.md file:
```python

from faster_whisper import WhisperModel, BatchedInferencePipeline

model = WhisperModel("turbo", device="cpu", compute_type="int8")
batched_model = BatchedInferencePipeline(model=model)
segments, info = batched_model.transcribe("audio.mp3", batch_size=16)

for segment in segments:
    print("[%.2fs -> %.2fs] %s" % (segment.start, segment.end, segment.text))
    

```

When I instead use the non-batched versions from the README, then I get many lines of output corresponding to lots of audio transcription, but the batched only gives me the first few seconds.  

Why is that?

**Comment by toanhuynhnguyen** [comment]
@drscotthawley  You can try adjusting the param "vad_parameters":
![image](https://github.com/user-attachments/assets/beb7b2c7-83e7-47b5-a3c4-4928c4eaef59)
